tests/puppeteer/campaigns/data/demo/sqlTables.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  Tables: {
+    ps_access: {
+      name: 'ps_access',
```

Comment:
the name look useless, use the key instead, wdyt?

Reply:
Done :)